### PR TITLE
Password inputs validation fixes

### PR DESCRIPTION
**Pull request by DrzikFilip**

### CODE REVIEW

File: resources/views/form.blade.php [hunk 1]
```diff
@@ -21,16 +21,16 @@
...
     <div :class="isFormLocalized ? 'col-md-4' : 'col-md-9 col-xl-8'">
-        <input type="password" v-model="form.{{ $col['name'] }}" v-validate="'{{ implode('|', $col['frontendRules']) }}'" @input="validate($event)" class="form-control" :class="{'form-control-danger': errors.has('{{ $col['name'] }}'), 'form-control-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid}" id="{{ $col['name'] }}" name="{{ $col['name'] }}" placeholder="{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}">
+        <input type="password" v-model="form.{{ $col['name'] }}" v-validate="'{{ implode('|', collect($col['frontendRules'])->reject(function($rule) use ($col) { return $rule === 'confirmed:'.$col['name'].'_confirmation';})->toArray()) }}'" @input="validate($event)" class="form-control" :class="{'form-control-danger': errors.has('{{ $col['name'] }}'), 'form-control-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid}" id="{{ $col['name'] }}" name="{{ $col['name'] }}" placeholder="{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}" ref="{{ $col['name'] }}">
```
Comment (dejwCake) [inline resources/views/form.blade.php]: This looks like a mistake: $rule === 'confirmed:'.$col['name'].'_confirmation'; Should be $rule === 'confirmed:'.$col['name'];

File: resources/views/templates/profile/form.blade.php [hunk 2]
```diff
@@ -1,15 +1,14 @@
...
-                                <input type="password" v-model="form.{{ $col['name'] }}" v-validate="'{{ implode('|', $col['frontendRules']) }}'" @input="validate($event)" class="form-control" :class="{'form-control-danger': errors.has('{{ $col['name'] }}'), 'form-control-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid}" id="{{ $col['name'] }}" name="{{ $col['name'] }}" placeholder="{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}">
+                                <input type="password" v-model="form.{{ $col['name'] }}" v-validate="'{{ implode('|', collect($col['frontendRules'])->reject(function($rule) use ($col) { return $rule === 'confirmed:'.$col['name'].'_confirmation';})->toArray()) }}'" @input="validate($event)" class="form-control" :class="{'form-control-danger': errors.has('{{ $col['name'] }}'), 'form-control-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid}" id="{{ $col['name'] }}" name="{{ $col['name'] }}" placeholder="{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}" ref="{{ $col['name'] }}">
```
Comment (dejwCake) [inline resources/views/templates/profile/form.blade.php]: This looks like a mistake: $rule === 'confirmed:'.$col['name'].'_confirmation'; Should be $rule === 'confirmed:'.$col['name'];

File: resources/views/templates/profile/password/form.blade.php [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+                            <label for="{{ $col['name'] }}" class="col-form-label text-md-right" :class="isFormLocalized ? 'col-md-4' : 'col-md-3'">{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}</label>
+                            <div :class="isFormLocalized ? 'col-md-4' : 'col-md-9 col-xl-7'">
+                                <input type="password" v-model="form.{{ $col['name'] }}" v-validate="'required|{{ implode('|', collect($col['frontendRules'])->reject(function($rule) use ($col) { return $rule === 'confirmed:'.$col['name'].'_confirmation';})->toArray()) }}'" @input="validate($event)" class="form-control" :class="{'form-control-danger': errors.has('{{ $col['name'] }}'), 'form-control-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid}" id="{{ $col['name'] }}" name="{{ $col['name'] }}" placeholder="{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}" ref="{{ $col['name'] }}">
```
Comment (dejwCake) [inline resources/views/templates/profile/password/form.blade.php]: This looks like a mistake: $rule === 'confirmed:'.$col['name'].'_confirmation'; Should be $rule === 'confirmed:'.$col['name'];

File: resources/views/templates/user/form.blade.php [hunk 4]
```diff
@@ -21,15 +21,15 @@
...
     <div :class="isFormLocalized ? 'col-md-4' : 'col-md-9 col-xl-7'">
-        <input type="password" v-model="form.{{ $col['name'] }}" v-validate="'{{ implode('|', $col['frontendRules']) }}'" @input="validate($event)" class="form-control" :class="{'form-control-danger': errors.has('{{ $col['name'] }}'), 'form-control-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid}" id="{{ $col['name'] }}" name="{{ $col['name'] }}" placeholder="{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}">
+        <input type="password" v-model="form.{{ $col['name'] }}" v-validate="'{{ implode('|', collect($col['frontendRules'])->reject(function($rule) use ($col) { return $rule === 'confirmed:'.$col['name'].'_confirmation';})->toArray()) }}'" @input="validate($event)" class="form-control" :class="{'form-control-danger': errors.has('{{ $col['name'] }}'), 'form-control-success': this.fields.{{ $col['name'] }} && this.fields.{{ $col['name'] }}.valid}" id="{{ $col['name'] }}" name="{{ $col['name'] }}" placeholder="{{'{{'}} trans('admin.{{ $modelLangFormat }}.columns.{{ $col['name'] }}') }}" ref="{{ $col['name'] }}">
```
Comment (dejwCake) [inline resources/views/templates/user/form.blade.php]: This looks like a mistake: $rule === 'confirmed:'.$col['name'].'_confirmation'; Should be $rule === 'confirmed:'.$col['name'];

File: resources/views/form.blade.php (hunk 1 above)
Comment (DrzikFilip) [reply resources/views/form.blade.php]: Thanks, it is fixed in next commit.

File: resources/views/templates/profile/form.blade.php (hunk 2 above)
Comment (DrzikFilip) [reply resources/views/templates/profile/form.blade.php]: Fixed in next commit.

File: resources/views/templates/profile/password/form.blade.php (hunk 3 above)
Comment (DrzikFilip) [reply resources/views/templates/profile/password/form.blade.php]: Fixed in next commit.

File: resources/views/templates/user/form.blade.php (hunk 4 above)
Comment (DrzikFilip) [reply resources/views/templates/user/form.blade.php]: Fixed in next commit.

**Pull request merged**